rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -50,54 +40,32 @@ At SSW we find the greatest overall value for expense reimbursements is to use t
  **Employees:** 
 
 
-
-
-  1. Using the Xero Expense mobile app, submit a photo of the receipt. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
-
-
-
-
-
-
-
+1. Open the Xero Expense mobile app and submit a photo of the receipt for reimbursement or filing. If the receipt was sent via email, you can still submit a photo or screenshot of the email receipt
 
 ![](2020-11-25_15-44-09.png)
 
-
  **Figure: Xero Expense app submit a photo** 
 
 
 
-
-
 2. Either allow the app to analyse the details manually or enter the details yourself (if you are strapped for time, you can take a photo and come back to it or hold on to the receipt and attend to it when possible)
 
 
-
-
 ![](e.jpg)
 
 
  **Figure: Details required of a submitted receipt** 
```

Comment:
Put this in the square brackets

Reply:
Done

Reply:
Please remove this line as the square brackets will make it show up on the live site.

Reply:
Done